source/renderer/app/containers/loading/SystemTimeErrorPage.js

```diff
@@ -13,25 +13,25 @@ export default class SystemTimeErrorPage extends Component<Props> {
 
   render() {
     const { stores } = this.props;
-    const {
-      localTimeDifference,
-      forceCheckTimeDifferenceRequest,
-      forceCheckLocalTimeDifference,
-      ignoreSystemTimeChecks,
-    } = stores.networkStatus;
+    // const {
+    //   localTimeDifference,
+    //   forceCheckTimeDifferenceRequest,
+    //   forceCheckLocalTimeDifference,
+    //   ignoreSystemTimeChecks,
+    // } = stores.networkStatus;
```

Comment:
@disassembler Can we remove this?

Reply:
@tomothespian no. I have left this logic in the code on purpose as we will probably re-enable this feature once we get NTP service support on the BE.